.travis.yml

```diff
@@ -11,6 +17,11 @@ jobs:
       script: sbt headerCheck test:headerCheck scalafmtCheck test:scalafmtCheck +test
     - stage: publish
       script: sbt +publish
+    - stage: validate-publish
+      script:
+        # Ensure that publishing works and produces valid artifacts
+        - sbt +publishLocal
+        - find . -name *.pom -exec xmllint --noout -schema http://maven.apache.org/xsd/maven-4.0.0.xsd {} \;
```

Comment:
This is a nice to have, but we may remove it if we decide validating pom files is too much.